modular_skyrat/modules/cellguns/code/medigun_research.dm

```diff
@@ -1,176 +1,147 @@
+#define RND_SUBCATEGORY_WEAPONS_MEDICALAMMO "/Medical Ammunition"
+#define RND_MEDICALAMMO_UTILITY " (Utility)"
+
+//Upgrade Kit//
+/datum/design/medigun_speedkit
+	name = "VeyMedical CWM-479 upgrade kit"
+	desc = "An upgrade kit for the VM CWM-479 to have a higher-capacity internal cell, with increased recharger throughput."
```

Comment:
You can probably afford to keep the full name to prevent confusion for this.
```suggestion
	desc = "An upgrade kit for the VeyMedical CWM-479 to have a higher-capacity internal cell, with increased recharger throughput."
```